<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emitterSettings</class>
 <widget class="QWidget" name="emitterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Projectile</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ProjectileCombo" name="cbProjectile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Emitter Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCopyEmitter">
       <property name="text">
        <string>Copy Emitter Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPasteEmitter">
       <property name="text">
        <string>Paste Emitter Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Spawn per step</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbSpawnPerStep"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Delay (ms)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbProjectileDelay">
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Start Angle</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="sbStartAngle">
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Angle Step</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="sbAngleStep">
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="sbSpeed">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Lifetime</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="sbLifetime">
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="sbOffsetX">
       <property name="minimum">
        <double>-500.000000000000000</double>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbOffsetY">
       <property name="minimum">
        <double>-500.000000000000000</double>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectileCombo</class>
   <extends>QComboBox</extends>
   <header>widgets/projectilecombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
